app/backend/controllers/login-signup.js

```diff
@@ -28,8 +28,10 @@ module.exports.signup = async (request, response) => {
 
   if(!request.body.googleId){
 
-    if(!request.body.password)
+    if(!request.body.password){
       response.status(400).send({ errmsg: 'Password is required in the request body' })
+      return
```

Comment:
It works as it is now I assume, but using return in controllers isn't a good practice, as we might want to add some other controllers to the endpoints where we used that one and those return statements might cause some hard-to-trace bugs.